Library/Homebrew/language/python.rb

```diff
@@ -115,13 +116,9 @@ module Virtualenv
       def self.included(base)
         base.class_eval do
           resource "homebrew-virtualenv" do
-            if MacOS.version > :mojave
-              url PYTHON_VIRTUALENV_URL
-              sha256 PYTHON_VIRTUALENV_SHA256
-            else
-              url PYTHON_VIRTUALENV_URL_MOJAVE
-              sha256 PYTHON_VIRTUALENV_SHA256_MOJAVE
-            end
+            url "https://files.pythonhosted.org/packages/11/74" \
+                "/2c151a13ef41ab9fb43b3c4ff9e788e0496ed7923b2078d42cab30622bdf/virtualenv-16.7.4.tar.gz"
+            sha256 "94a6898293d07f84a98add34c4df900f8ec64a570292279f6d91c781d37fd305"
```

Comment:
Probably not in this PR, but we need to consider using a newer version of `virtualenv`. 
Versions prior to `v20.0.0` belong to the "legacy" branch (the latest one being 16.7.10 -- maybe we could use it here even now) and the latest stable is: v20.0.31

https://virtualenv.pypa.io/en/latest/changelog.html

Reply:
Using v20 is complicated as it requires a bunch of dependencies.

16.7.10 should be trivial to update to however.